Client/Frontend/Fakespot/Views/FakespotReviewQualityCardView.swift

```diff
@@ -69,18 +85,24 @@ final class FakespotReviewQualityCardView: UIView, Notifiable, ThemeApplicable {
         label.numberOfLines = 0
         label.adjustsFontForContentSizeCategory = true
         label.clipsToBounds = true
-        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body,
+        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .subheadline,
                                                             size: UX.labelFontSize)
     }
 
     private lazy var subHeadlineLabel: UILabel = .build { label in
-        label.text = .Shopping.ReviewQualityCardSubHeadlineLabel
+        let text = String.Shopping.ReviewQualityCardSubHeadlineLabel
+        let normalFont = DefaultDynamicFontHelper.preferredFont(withTextStyle: .subheadline,
+                                                                size: UX.labelFontSize)
+        let boldFont = DefaultDynamicFontHelper.preferredFont(withTextStyle: .subheadline,
+                                                              size: UX.labelFontSize,
+                                                              weight: .semibold)
+        label.attributedText = text.attributedText(boldPartsOfString: ["letter grade"],
```

Comment:
This will not work for any other language - so this string needs to be localized as well.

Reply:
omg nice catch!